packages/devtools_app/lib/src/debugger/program_explorer_controller.dart

```diff
@@ -98,6 +98,7 @@ class ProgramExplorerController extends DisposableController
       return;
     }
     _selectScriptNode(script, _rootObjectNodes.value);
+    // ignore: invalid_use_of_visible_for_testing_member, invalid_use_of_protected_member
```

Comment:
why are these ignores necessary? These should be calling ListValueNotifier.notifyListeners(), which isn't marked visible for testing nor protected.

Reply:
interesting, now they are not needed
may be something changed in master?
removed